### PR TITLE
Library removal

### PR DESCRIPTION
**Pull request by designer2k2**
Hello,
please remove the EMUcanT4 library https://github.com/designer2k2/EMUcanT4 (it is obsolete)
thanks!

### CODE REVIEW

**Comment by per1234** [comment]
Hello @designer2k2. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by umbynos — APPROVED** [review]

**Comment by umbynos** [comment]
Done in:
```
2023/01/10 16:52:55 Loaded 5483 libraries from DB
Removing EMUcanT4
Success!
```

**Pull request merged**
